meta-mender-core/classes/mender-part-images.bbclass

```diff
@@ -66,9 +66,12 @@ mender_part_image() {
     echo "device_type=${MENDER_DEVICE_TYPE}" > "${WORKDIR}/data/mender/device_type"
     chmod 0444 "${WORKDIR}/data/mender/device_type"
 
-    dd if=/dev/zero of="${WORKDIR}/data.${ARTIFACTIMG_FSTYPE}" count=0 bs=1M seek=${MENDER_DATA_PART_SIZE_MB}
-    mkfs.${ARTIFACTIMG_FSTYPE} -F "${WORKDIR}/data.${ARTIFACTIMG_FSTYPE}" -d "${WORKDIR}/data" -L data
-    install -m 0644 "${WORKDIR}/data.${ARTIFACTIMG_FSTYPE}" "${DEPLOY_DIR_IMAGE}/"
+    # Use rootfs FSTYPE unless stated otherwise
+    ARTIFACTIMG_DATA_FSTYPE ?= "${ARTIFACTIMG_FSTYPE}"
+
+    dd if=/dev/zero of="${WORKDIR}/data.${ARTIFACTIMG_DATA_FSTYPE}" count=0 bs=1M seek=${MENDER_DATA_PART_SIZE_MB}
+    mkfs.${ARTIFACTIMG_FSTYPE} -F "${WORKDIR}/data.${ARTIFACTIMG_DATA_FSTYPE}" -d "${WORKDIR}/data" -L data
```

Comment:
Shouldn't this be invoced as mkfs.${ARTIFACTIMG_DATA_FSTYPE}?

Reply:
Thanks! I'll force push the change. It's been lost during the rebase on current master. Initially the work had been based on rocko branch. Thanks for noticing.